spec/features/saml/loa3_sso_spec.rb

```diff
@@ -4,7 +4,7 @@
   include SamlAuthHelper
   include IdvHelper
 
-  def perform_id_verification_with_usps_without_confirming_code_then_sign_out(user)
+  def perform_id_verification_with_usps_without_confirming_code(user)
     saml_authn_request = auth_request.create(loa3_with_bundle_saml_settings)
     visit saml_authn_request
     sign_in_live_with_2fa(user)
```

Comment:
I know this was like this before this PR, but to make sure we're testing things exactly like the user would experience, can we change `sign_in_live_with_2fa(user)` to this instead?
```ruby
click_link t('links.sign_in')
fill_in_credentials_and_submit(user.email, user.password)
click_submit_default
```
The difference is that the latter preserves the `request_id` in the URL, whereas the former doesn't, and we have logic in the app that does different things depending on whether or not the `request_id` is present.